openjdk/ProblemList_openjdk8.txt

```diff
@@ -171,10 +175,16 @@ java/nio/channels/ServerSocketChannel/AdaptServerSocket.java	https://github.com/
 java/nio/channels/DatagramChannel/SendToUnresolved.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/827	linux-all
 java/nio/channels/Selector/RacyDeregister.java	https://bugs.openjdk.java.net/browse/JDK-8161083	aix-all
 java/nio/channels/AsyncCloseAndInterrupt.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/593	aix-all
+java/nio/channels/FileChannel/Transfers.java    https://github.com/AdoptOpenJDK/openjdk-tests/issues/1625   linux-aarch32
 sun/nio/ch/TestMaxCachedBufferSize.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1051 linux-s390x
-sun/nio/cs/OLD/TestIBMDB.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1051 linux-s390x
-java/nio/MappedByteBuffer/Truncate.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1052 linux-s390x
+sun/nio/cs/OLD/TestIBMDB.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1051 linux-s390x, linux-aarch32
```

Comment:
Can not have spaces between comma separated platforms in exclude files (or jtreg will not handle excludes correctly), some examples below also.